### PR TITLE
Lobby platform

### PR DESCRIPTION
**Pull request by polina-tsvilodub**
Start review of Tobias' contributions on the lobby platform docs. Sing who will also be reviewing contributions to this branch is assigned as reviewer.

### CODE REVIEW

File: _posts/2022-03-17-lobby.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
...pt files interpretable for browsers Webpack is used to compile and bundle them into a single optimized JavaScript file. For the configuration of Webpack we used the [webpack-boilerplate](https://github.com/taniarascia/webpack-boilerplate) template by Tania Rascia which already comes with a local development server and a build process to optimize bundling for production. The Webpack loader ts-loader was added so that Webpack is able to compile and bundle TypeScript files. Additional dependencies like `xstate` and `socket.io-client` were added to the package.json as well. For additional information on Webpack refer to [Tania’s tutorial](https://www.taniarascia.com/how-to-use-webpack/) or the [official Webpack documentation](https://webpack.js.org/guides/). The Webpack production built process is also run when the application is deployed to Heroku where the application is hosted on the free tier. The necessary steps for deployment are somewhat documented by [this commit](https://github.com/TobiObeck/escapeling-lobby/commit/8bbeab11db2a4a3d7402ed9a5490994a56880d00). The lobby is publicly available here: [https://escapeling-lobby.herokuapp.com/.](https://escapeling-lobby.herokuapp.com/)
```
[the remark this answers is not in the document]
Comment (polina-tsvilodub) [reply _posts/2022-03-17-lobby.md:59]: This is the first section where package names etc are written using `this markup`. Maybe one could use consistent styling throughout the entire page.

[the remark this answers is not in the document]
Comment (polina-tsvilodub) [reply _posts/2022-03-17-lobby.md:59]: Overall, great job! I think you provide sufficient technical details, while not overwhelming the reader. The text is clear and concise, though the sentences are sometimes rather long ;) thanks for your work Tobi!

**Review by polina-tsvilodub — COMMENTED** [review]
Finished review of Tobi's contribution.

File: README.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 ---
 
-### Note for Semester Three
+### Note for Semester Four
 
-The `master` branch is now updated up to the third semester of work on this project. The documentation site is currently being deployed from that branch.
+The `master` branch is now updated up to the fourth semester of work on this project. The documentation site is currently being deployed from that branch.
 
-For current and future reference, there are two archive branches: `semester-two-archive` and `semester-three-archive`.
+For current and future reference, there are three archive branches: `semester-two-archive` and `semester-three-archive` and 'semester-four'
```
[the remark this answers is not in the document]
Comment (TobiObeck) [reply README.md:9]: I guess it will be in a few days, once we don't do anything anymore 😅

**Pull request merged**